CMakeLists.txt

```diff
@@ -148,12 +148,12 @@ find_library(FOUND_UNWIND_LIB unwind)
 set(WARNING_FLAGS -Wall -Wpedantic -Wsign-compare -Wreturn-type)
 
 # kvrocks objects target
-file(GLOB KVROCKS_SRCS src/*.cc)
+file(GLOB_RECURSE KVROCKS_SRCS src/*.cc)
 list(FILTER KVROCKS_SRCS EXCLUDE REGEX src/main.cc)
 
 add_library(kvrocks_objs OBJECT ${KVROCKS_SRCS})
 
-target_include_directories(kvrocks_objs PUBLIC src ${PROJECT_BINARY_DIR})
+target_include_directories(kvrocks_objs PUBLIC src src/common ${PROJECT_BINARY_DIR})
```

Comment:
What's the reason of this change?

Reply:
Since header files in `common` is frequently used, so i.e. it can be just `"status.h"` rather than `"common/status.h"`.